app/main/views/g_cloud.py

```diff
@@ -544,6 +544,62 @@ def end_search(framework_framework, project_id):
     )
 
 
+@direct_award.route(
+    '/<string:framework_framework>/projects/<int:project_id>/did-you-award-contract',
+    methods=['GET', 'POST']
+)
+def did_you_award_contract(framework_framework, project_id):
+    all_frameworks = data_api_client.find_frameworks().get('frameworks')
+    framework = framework_helpers.get_latest_live_framework(all_frameworks, framework_framework)
+
+    # Get the requested Direct Award Project.
+    project = data_api_client.get_direct_award_project(project_id=project_id)['project']
+
+    form = DidYouAwardAContractForm()
+    form.did_you_award_a_contract.options = [{"label": label, "value": value}
+                                             for value, label in form.did_you_award_a_contract.choices]
+
+    if form.validate_on_submit():
+        if form.did_you_award_a_contract.data == 'still-assessing':
+            return redirect(url_for('.view_project',
+                                    framework_framework=framework_framework,
+                                    project_id=project_id))
+        elif form.did_you_award_a_contract.data == 'yes':
+            return redirect(url_for('.which_service_won_contract',
+                                    framework_framework=framework_framework,
+                                    project_id=project_id))
+        elif form.did_you_award_a_contract.data == 'no':
+            return redirect(url_for('.why_didnt_you_award_contract',
+                                    framework_framework=framework_framework,
+                                    project_id=project_id))
+        else:
+            # this else should never be reached
+            abort(500, "Unexpected answer to 'Did you award contract?'")
```

Comment:
I know Dilwoar mentioned this already, I think if you want to leave it in I'd be find with not bothering to have a message, `abort(500)` would be fine.